splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/util/uccConfigurationValidators.js

```diff
@@ -0,0 +1,255 @@
+import schema from '../schema/schema.json';
+import {Validator} from 'jsonschema';
+import * as _ from 'lodash';
```

Comment:
If possible we should avoid such large dependency.

Reply:
Okay, I will remove it after discussing it in the scrum